docker/volumeplugin/rancher.go

```diff
@@ -50,7 +50,7 @@ func NewRancherState(driver string, client *client.RancherClient) (*RancherState
 }
 
 func (r *RancherState) IsCreated(name string) (bool, error) {
-	_, vol, err := r.Get(name)
+	_, vol, err := r.GetAny(name)
```

Comment:
Yup. We can get rid of the last line code  too, maybe that is the better solution. User Get() at the first line, then return return true, nil